packages/flutter/lib/src/cupertino/segment_control.dart

```diff
@@ -0,0 +1,191 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/painting.dart';
+
+class SegmentControlItem {
+  final String title;
+  final Widget content;
+
+  SegmentControlItem(this.title, this.content);
```

Comment:
please see our style guide and see other code in the framework for unwritten conventions. for example, we put constructors first, we use `:` instead of `=` for default argument values, we put spaces around the named arguments inside `{}` in constructors, createState should use `=>` and have its return type be explicit...

Reply:
Would prefer named arguments rather than positional ones to make caller code clearer.

I think the linter would suggest const constructors for immutable classes too